TutorLizard.BusinessLogic/Data/IStudentDataAccess.cs

```diff
@@ -1,5 +1,12 @@
-﻿namespace TutorLizard.BusinessLogic.Data;
+﻿using TutorLizard.BusinessLogic.Models;
+
+namespace TutorLizard.BusinessLogic.Data;
 
 public interface IStudentDataAccess
 {
+    AdRequest CreateAdRequest(int adId, object value, bool isAccepted, string message);
+    ScheduleItemRequest CreateScheduleItemRequest(int scheduleItemId, int value);
```

Comment:
Rename `value` to `studentId`.
I'd change the signature to what already exists in `DataAccess`:
`public ScheduleItemRequest CreateScheduleItemRequest (int scheduleItemId, int userId, bool isAccepted)`
You always want isAccepted to be false, but you can just provide it each time you call it.